testsuite/tests/tool-ocamlopt-save-ir/check_for_pack.compilers.reference

```diff
@@ -1,2 +1,2 @@
 File "check_for_pack.cmir-linear", line 1:
-Error: This input file cannot be compiled with -for-pack foo: it was generated without -for-pack.
+Error: This input file cannot be compiled with -for-pack foo : it was generated without -for-pack.
```

Comment:
I wonder why we have a change here, and whether it should be considered a minor regressions.

Reply:
That was a wandering white space, that I have fixed.